src/Compute/Compute.Test/ScenarioTests/VirtualMachineExtensionTests.ps1

```diff
@@ -2030,7 +2030,7 @@ function Test-VirtualMachineADDomainExtensionDomainJoin
         # Set ADDomain extension
         Assert-ThrowsContains { Set-AzVMADDomainExtension -ResourceGroupName $rgname -Location $loc -VMName $vmname -Name $extname `
             -DomainName $domainName -Credential $cred2 -OUPath $ouPath -JoinOption 3 -Restart; } `
-            "occurred while joining Domain";
+            "occured while joining Domain";
```

Comment:
Looks like this reverted

Reply:
Correct, the test was failing without this change -- I think we'll be fine if it's misspelled since it's just test code and should be updated when it's re-recorded in the future.